resources/assets/js/components/GroupEvents.vue

```diff
@@ -36,7 +36,7 @@
               </b-tbody>
             </b-table-simple>
             <div class="text-center" v-if="limit">
-              <b-btn variant="link" :href="'/party/group/' + groupId">
+              <b-btn variant="link" :href="'/party/group/' + idgroups">
```

Comment:
Noticing this change (groupId -> idgroups) a lot - just a refactoring for consistency with the DB column name?

Reply:
Yes, exactly.